tests/integration/solidity_contracts/EFTests/test_sstore.py

```diff
@@ -0,0 +1,95 @@
+import asyncio
+import logging
+
+import pytest
+from starkware.starknet.testing.contract import StarknetContract
+
+from tests.utils.helpers import (
+    hex_string_to_bytes_array,
+    private_key_from_hex,
+)
+from starkware.starknet.public.abi import get_storage_var_address
+from starkware.starknet.testing.contract_utils import gather_deprecated_compiled_class
+from tests.utils.uint256 import hex_string_to_uint256
+
+logger = logging.getLogger()
+
+
+@pytest.mark.asyncio
+@pytest.mark.EF_TEST
+@pytest.mark.SSTORE
+class TestSSTORE:
+    @pytest.mark.skip("TODO: deploy_contract_account should also increment nonce")
+    async def test_InitCollision_d0g0v0_Shanghai(
+        self,
+        starknet,
+        get_starknet_address,
+        deploy_eoa,
+        kakarot: StarknetContract,
+    ):
+        evm_contract_address = 0x6295EE1B4F6DD65047762F924ECD367C17EABF8F
+        starknet_contract_address = get_starknet_address(evm_contract_address)
+
+        # Set storage
+        storage = {"0x01": "0x01"}
+        storage_key = get_storage_var_address(
+            "storage_", *hex_string_to_uint256("0x01")
+        )
+        keys = (storage_key, storage_key + 1)  # (low, high)
+        values = hex_string_to_uint256(storage["0x01"])
+        for key, value in zip(keys, values):
+            await starknet.state.state.set_storage_at(
+                starknet_contract_address, key, value
+            )
+
+        # Check initial storage
+        storage_initial = await asyncio.gather(
+            *(
+                starknet.state.state.get_storage_at(starknet_contract_address, key)
+                for key in keys
+            )
+        )
+        assert storage_initial == list(hex_string_to_uint256(storage["0x01"]))
```

Comment:
looks like this is a perfect fit for the fixture no?

Reply:
Yeah you're right, it can be handy for us to initialize the empty contract accounts 🙂 Does it mean we can only do this for contracts with no bytecode? To my understanding, we need to actually deploy them using `create_account_with_bytecode_and_storage` to actually use it right?